pkgs/data/icons/candy-icons/default.nix

```
@@ -0,0 +1,26 @@
+{ pkgs }:
+
+pkgs.stdenv.mkDerivation {
+
+  pname = "candy-icons";
+  version = "0.6.8";
+
+  nativeBuildInputs = [ pkgs.gtk3 ];
+
+  src = pkgs.fetchurl {
+    url = "https://github.com/EliverLara/candy-icons/archive/37b4566b4bfcdad99131a236ef1fc74db89a302e.zip";
+    sha256 = "fgey2MduS9alUKiFhZr6KTKhYa68zh6LzEIcg/amrOo=";
+  };
+
+  dontUnpack = true;
+
+  installPhase = ''
+    mkdir -p $out/share/icons
+    ${pkgs.unzip}/bin/unzip $src -d $out/share/icons/
+    mv $out/share/icons/candy-icons-* $out/share/icons/candy-icons
+
+    for theme in $out/share/icons/*; do
+      gtk-update-icon-cache --force $theme
+    done
+  '';
```

Since we're now using `fetchzip`, there's no need for unpacking. Also, every package should have `runHooks`s.

```suggestion
  installPhase = ''
    runHook preInstall
  
    mkdir -p $out/share/icons
    mv candy-icons-* $out/share/icons/candy-icons

    for theme in $out/share/icons/*; do
      gtk-update-icon-cache --force $theme
    done
    
    runHook postInstall
  '';
```